src/main/java/io/airlift/slice/FNVHash.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.airlift.slice;
+
+/**
+ * Reference implementation: https://tools.ietf.org/html/draft-eastlake-fnv-17#section-6
+ */
+public class FNVHash
```

Comment:
Capitalize as `FnvHash`. We do that to improve readability when many acronyms are used in a row (e.g., `HTTPURL` vs `HttpUrl`, `HTTPXMLRPC` vs `HttpXmlRpc`).